pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -167,7 +167,7 @@ func (o *capability) Execute(ctx context.Context, r capabilities.CapabilityReque
 	}
 	err := r.Inputs.UnwrapTo(&m)
 	if err != nil {
-		o.lggr.Warnf("could not unwrap method from CapabilityRequest, using default: %w", err)
+		o.lggr.Warnf("could not unwrap method from CapabilityRequest, using default: %v", err)
```

Comment:
If this is getting signaled because `err` isn't of type `error`, then a variable name change might be useful to differentiate.

Reply:
Nah because %w is just for constructing new errors, not formatting for a log message.

Reply:
For posterity, here is the discussion why %w should not be used in the logging formatting functions https://github.com/golang/go/issues/47641

Reply:
go vet will complain about some std lib cases, but not even all of those are caught until runtime:
https://go.dev/play/p/KQCLIA7ZBM3